pages/index.js

```diff
@@ -5,17 +5,26 @@ import SectionSeparator from '../components/SectionSeparator'
 import SpeakersSection from '../components/SpeakersSection'
 import TicketsSection from '../components/TicketsSection'
 import SponsorsSection from '../components/SponsorsSection'
+import { Element } from 'react-scroll'
 
 export default () => (
   <MainLayout>
     <HeroUnit />
     <SectionSeparator first />
-    <section id='tickets'><TicketsSection /></section>
+    <Element name='tickets'>
```

Comment:
The page should not be aware of the scrolling mechanism.

Ideally, only one file should be changed to allow smooth scrolling.